server/views/components/header-block/index.njk

```diff
@@ -0,0 +1,10 @@
+<header class="header-block {% for modifier in modifiers %}header-block--{{ modifier }} {% endfor %}">
```

Comment:
Again, composing CSS names is almost anti-BEM - and makes static analysis hard.
Could we not do something easier and just have extra classes which is more helpful for practical use?